src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -40,6 +36,19 @@ public CommandResult execute() {
         return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
     }
 
+    /**
+     *
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.
Not only the param and return, but also comment about "what this method does"